mappings/net/minecraft/client/particle/ParticleManager.mapping

```diff
@@ -0,0 +1,30 @@
+CLASS net/minecraft/class_75 net/minecraft/client/particle/ParticleManager
+	FIELD field_269 world Lnet/minecraft/class_18;
+	FIELD field_270 particles [Ljava/util/List;
+	FIELD field_271 textureManager Lnet/minecraft/class_76;
+	FIELD field_272 random Ljava/util/Random;
+	METHOD <init> (Lnet/minecraft/class_18;Lnet/minecraft/class_76;)V
+		ARG 1 world
+		ARG 2 textureManager
+	METHOD method_320 removeDeadParticles ()V
+	METHOD method_321 spawnBlockBreakingParticles (IIII)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 side
+	METHOD method_322 spawnBlockRemoveParticles (IIIII)V
```

Comment:
[addBlockBreakParticles](https://maven.fabricmc.net/docs/yarn-1.21.1+build.3/net/minecraft/client/particle/ParticleManager.html#addBlockBreakParticles(net.minecraft.util.math.BlockPos,net.minecraft.block.BlockState)) in Yarn

Reply:
done